mk/spksrc.service.use_alternate_tmpdir.dsm7

```diff
@@ -0,0 +1,32 @@
+
+# use DSM7 package specific tmp folder as alternate TMPDIR, this can be useful to avoid errors like:
+# Could not install packages due to an EnvironmentError: [Errno 28] No space left on device
+# as /tmp might be too small.
+# typically used to install a huge amount of wheels with pip (see homeassistant package)
+# TMPDIR is supported by pip https://github.com/pypa/pip/issues/4462 ('--build dir' is not supported anymore)
+
+TMPDIR=${SYNOPKG_PKGTMP}/tmp
```

Comment:
@hgy59 sorry for being late to the party.

You should drop the `/tmp` you are already there with `{SYNOPKG_PKGTMP}` 

```
{SYNOPKG_PKGTMP} = tmp-> /usr/local/packages/@apptemp/[package_name]
aka /var/packages/[package_name]/tmp
```

Reply:
Oh never mind you already fixed it 🙄